jekyll/_cci2/skip-build.md

```diff
@@ -59,38 +59,37 @@ If you push multiple commits at once, a single `[ci skip]` or `[skip ci]` will s
 
 ## Auto Cancelling a Redundant Build
 
-If you are frequently pushing changes to a branch, you increase the chances of builds queueing. This means you might have to wait for an older version of a branch to finish building before the most recent version builds.
+If you are frequently pushing changes to a branch, you increase the chances of queueing. This means you might have to wait for an older pipeline to finish building before the most recent version starts.
 
-To save time, you can configure CircleCI to automatically cancel any queued or running builds when a newer build is triggered on that same branch.
+To save time, you can configure CircleCI to automatically cancel any queued or running pipelines when a newer pipeline is triggered on that same branch.
 
-**Note:**
-Your project's default branch will never auto-cancel builds. This feature only applies to non-workflow builds, builds triggered by pushes to GitHub, or workflow builds that use pipelines.
+**Note:** Your project's default branch (usually `master`) will never auto-cancel builds. 
 
-### Steps to Enable Auto-Cancel for New Builds Triggered by Pushes to GitHub without Workflows
+### Steps to Enable Auto-Cancel for Pipelines Triggered by Pushes to GitHub or the API
 {:.no_toc}
 
-1. In the CircleCI application, go to your project's settings by clicking the gear icon next to your project.
+**Notes:** It is important to carefully consider the impact of enabling the auto-cancel feature, for example, if you have configured automated deployment jobs on non-default branches. Auto-cancelling workflows requires enabling pipelines.
```

Comment:
~ I think we can drop `Auto-cancelling workflows requires enabling pipelines.`, everything in dot-com has Pipelines enabled.

If these docs are also used for Server then we can keep it but modify a bit.

Reply:
Ah yes I missed that, thanks!